### PR TITLE
Thomryan itdependsnetworks token tests

### PR DESCRIPTION
**Pull request by ThomasJRyan**
## Description

Resolving conflicts in #258

## Screenshots:

image
image

## Checklist:

- [ ] I have updated the changelog.
- [ ] I have updated the documentation (If applicable).
- [ ] I have added tests to cover my changes (If applicable).
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
-    Temp: FC PWM1    P6                Fan Speed 45%     25 Celsius
\ No newline at end of file
+    Temp: FC PWM1    P6                Fan Speed 45%     25 Celsius
+=======
+Router#### FAN 7 OUT ###show env all | include Sensor | P6 | P7
+Sensor List:  Environmental Monitoring 
+ Temp1            P6                Normal            40 Celsius
+ Temp: FC PWM1    P6                Fan Speed 45%     25 Celsius
+    
+>>>>>>> 17601e0f39749dead696ac035e340e4c6dad598d
```
Comment (ahersi2-cisco) [inline src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt:11]: maybe missed this?

File: src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt [hunk 2]
```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
```
Comment (LenzDong) [inline src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt:1]: please resolve conflict

**Review by LenzDong — APPROVED** [review]

**Review by ahersi2-cisco — APPROVED** [review]

File: src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt (hunk 2 above)
Comment (ThomasJRyan) [reply src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt]: Done

File: src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt (hunk 1 above)
Comment (ThomasJRyan) [reply src/genie/libs/parser/iosxe/asr1k/tests/ShowEnvironmentAllIncludeLocation/cli/equal/golden_output_output.txt]: Done

**Pull request merged**